test/pytorch_mnist/data/clean

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+rm data* -rf
```

Comment:
for this to work you need to change the position of arguments recursive/force '-rf' to $rm -rf data*